.coderabbit.yaml

```
@@ -0,0 +1,64 @@
+  include_patterns:
+    - "**/*.tsx"
+    - "**/*.ts" 
+    - "**/*.json"
+    - "**/*.md"
+    - "**/*.yaml"
+    - "**/*.yml"
+  
+  # Review focus areas for spiritual app
+  focus_areas:
+    - "Security vulnerabilities and API key exposure"
+    - "TypeScript strict mode compliance" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)